yarn.lock

```diff
@@ -0,0 +1,2052 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Is this file intentionally added to the repo?

Reply:
Well for those who use yarn it will speed up the install of the node_modules. Do you want me to remove it?

Reply:
Yeah, I think it should be removed since I don't use Yarn and it would get out of date.